src/features/dataset/DatasetNavSidebar.tsx

```diff
@@ -106,7 +106,8 @@ const DatasetNavSidebar: React.FC<DatasetNavSidebarProps> = ({ qriRef }) => {
           id='workflow-editor'
           icon='code'
           label='Workflow'
-          to={pathToWorkflowEditor(qriRef.username, qriRef.name)}
+          isLink={!isNewWorkflow}
+          to={isNewWorkflow ? '/workflow/new' : pathToWorkflowEditor(qriRef.username, qriRef.name)}
```

Comment:
since you've added `isLink`, we don't need to send a `/workflow/new` as a possible path, since `isLink == true` and `to == '/workflow/new'` are mutually exclusive.

Can just be:
```js
    isLink={!isNewWorkflow}
    to={pathToWorkflowEditor(qriRef.username, qriRef.name}
```

Reply:
Hey @ramfox , actually the link is used to determine if the text is highlighted (active). And it previously was not active which I think is sort of a bug. Let me know what you think